### PR TITLE
Fixed some defaults

### PR DESCRIPTION
**Pull request by knee-berts**

### CODE REVIEW

File: cli/pkg/config/tfvars_generator.go [hunk 1]
```diff
@@ -19,7 +19,11 @@ func GenerateTfvars(conf *Config) {
 	vars["ClustersProjectId"] = conf.ClustersProjectID
 	vars["GovernanceProjectId"] = conf.GovernanceProjectID
 	vars["ConfigSync"] = conf.ConfigSync
-	vars["PrivateEndpoint"] = conf.PrivateEndpoint
+	if conf.PrivateEndpoint == true {
```
Comment (askmeegs) [inline cli/pkg/config/tfvars_generator.go:22]: Could you add a comment explaining why this var gets flipped? Thanks!

File: cli/samples/default-config.yaml [hunk 2]
```diff
@@ -21,18 +21,16 @@ vpcConfig:
...
 - clusterName: "gke-central"
-  projectId: "my-project"
   numNodes: 3 
   clusterType: "public"
   machineType: "e2-standard-4"
   region: "us-central1" 
   zone: "us-central1-b"
   subnetName: "us-central1"
 - clusterName: "gke-east"
-  projectId: "my-project"
```
Comment (askmeegs) [inline cli/samples/default-config.yaml:32]: Why did you remove these?

File: cli/samples/multi-cluster.yaml [hunk 3]
```diff
@@ -0,0 +1,31 @@
+# sample-config.yaml
+terraformState: local  # local, cloud
```
Comment (askmeegs) [inline cli/samples/multi-cluster.yaml:2]: Did you intend to push this file?

**Review by askmeegs — CHANGES_REQUESTED** [review]
A few Qs!

File: cli/samples/multi-cluster.yaml (hunk 3 above)
Comment (knee-berts) [reply cli/samples/multi-cluster.yaml:2]: I did. I wanted to have a folder of examples for folks to use. Should we put it somewhere else?

File: cli/samples/default-config.yaml (hunk 2 above)
Comment (knee-berts) [reply cli/samples/default-config.yaml:32]: Good catch. lmk fix

Comment (knee-berts) [reply cli/samples/default-config.yaml:32]: Oh wait, I removed those because they are not used. All cluster projects are derived from the base vars "clustersProjectId"

File: cli/pkg/config/tfvars_generator.go (hunk 1 above)
Comment (knee-berts) [reply cli/pkg/config/tfvars_generator.go:22]: Yep, great idea!

File: cli/samples/default-config.yaml (hunk 2 above)
Comment (askmeegs) [reply cli/samples/default-config.yaml:32]: gotcha, then should we remove the cluster config field `projectId` ? that's still in config.go

File: cli/samples/multi-cluster.yaml (hunk 3 above)
Comment (askmeegs) [reply cli/samples/multi-cluster.yaml:2]: no this is fine! just making sure you didn't push it by accident

File: cli/samples/default-config.yaml (hunk 2 above)
Comment (knee-berts) [reply cli/samples/default-config.yaml:32]: Done, thanks!
